server/static/js/index/home_page_preloader.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
+const DEVICE_PRELOADER = '#table_device_preloader';
+const USERS_PRELOADER = '#table_users_preloader';
+
+// The function is intended to execute the script when sending an AJAX request.
+function beforeLoadTableDevice () {
```

Comment:
Function names should give a hint about what they do. Consider renaming it with this in mind.
In this case it can be something like showLoaderInDevicesTable()